CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ if (SPT3G_VERSION)
 endif()
 
 # Ignore known warnings
-target_compile_options(spt3g INTERFACE -Wno-unknown-warning-option -Wno-unused -Wno-unused-result -Wno-deprecated-register -Wno-sign-compare -Wno-constant-conversion -Wno-self-assign-overloaded -Wno-deprecated-declarations)
+target_compile_options(spt3g INTERFACE -Wno-unknown-warning-option -Wno-unused -Wno-unused-result -Wno-deprecated-register -Wno-sign-compare -Wno-constant-conversion -Wno-self-assign-overloaded -Wno-deprecated-declarations -Wno-maybe-uninitialized)
```

Comment:
What place in the code is this fixing? I'd prefer to fix the code, since this is (often) an actually useful warning being turned off.

Reply:
At least one place was I believe this, which is out of our control. https://github.com/boostorg/python/issues/180